SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -228,6 +212,8 @@ void USpatialActorChannel::Init(UNetConnection* InConnection, int32 ChannelIndex
 	LastPositionSinceUpdate = FVector::ZeroVector;
 	TimeWhenPositionLastUpdated = 0.0f;
 	AuthorityReceivedTimestamp = 0;
+	MigrationFrame = 0;
```

Comment:
Why does this live in actor channel?
It seems cleaner to have meta-entity stuff not tightly coupled to unreal stuff. Especially as actor channels don't always exist and it's hard to tell what the state of all entities is at this point.

Reply:
For lack of a better place to put this information. If you have an idea where that could live ?
This is duplicated information amongst all actors of a group, so it could belong to an object representing the group eventually.